### ISSUE TITLE
[FEATURE] Spoofing Story Views

### ISSUE THREAD

**Issue by advarsity** [eni9889/SC-PP-ISSUES#89]
Just like how you can spoof your followings on Instagram++, there should be a button next to your story that lets you edit the number of views. Would be super cool.

**Comment by Loganreese12** [comment]
Agreed, a way to locally change the number of views your story has as well as the number of screenshots would be great.
